tsconfig.base.json

```diff
@@ -17,7 +17,6 @@
     "noUnusedParameters": true,
     "noUnusedLocals": true,
     "forceConsistentCasingInFileNames": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
Do we want to turn this back on in the cjs file?

This is only an error thing that we're changing, so we don't think that turning this off will add unecessary `requires` calls in the CJS build right?